app/grandchallenge/components/templates/components/partials/example_download_link.html

```diff
@@ -0,0 +1,8 @@
+{% load json %}
+
+{% if object.json_kind_example %}
+    Example usage: <a href="data:text/plain;charset=utf-8,{{ object.json_kind_example.value|json_dumps|urlencode }}"
+        download="example.json">
```

Comment:
If there are a lot of files to download, would it make sense to incorporate the interface slug in the download filename somehow?

Reply:
sure, good idea. It should be possible.

Reply:
Done